### ISSUE TITLE
Plane Takeoff & Plane Landing Icons

### ISSUE THREAD

**Issue by jis3r** [lucide-icons/lucide#781]
## Icon Request

* Icon name: Plane Takeoff & Plane Landing
* Use case: For travel websites or airtraffic-related topics
* _Screenshots_ of similar icons (Font Awesome):
![image](https://user-images.githubusercontent.com/72663107/182880640-3d54b3f5-8890-498b-9eba-65c4e8858256.png)
![image](https://user-images.githubusercontent.com/72663107/182880542-3933887f-6170-4151-860d-96c3ad994ff0.png)

**Comment by jis3r** [comment]
I started to work on them, but am not sure if the results are satisfying yet. Here a screenshot from figma:
![image](https://user-images.githubusercontent.com/72663107/182881412-4c7a07d5-8062-4cd1-9119-0d0a9043ba26.png)

**Comment by jis3r** [comment]
_Accidentally closed the issue, sorrry._

I already started to work on the icons, but I am not perfectly happy with the results. Here a screenshot from Figma:
![image](https://user-images.githubusercontent.com/72663107/182882236-5098c9f5-8049-4d33-bb48-5da79f7c3060.png)

Unless you think they are already good, I would be happy about other submissions! :)

**Comment by karsa-mistmere** [comment]
I think the tail part definitely needs to be a bit less dense than it currently is and some corners could also do with a bit of rounding.

This is my take based on yours (but I'm not quite happy yet with the level of pixel perfection I've managed to achieve):
![image](https://user-images.githubusercontent.com/17746067/182904683-4758af03-d438-4fae-8b91-984ff9989650.png)
![plane](https://user-images.githubusercontent.com/17746067/182904922-c16310ba-97ac-4b80-b24b-509f5ac279f9.png)

**Comment by jis3r** [comment]
You definitely got a point there, I too think that yours look better. I found it hard to get the plane good looking because of the limited level of detail due to the relatively thick line with 2px.

**Comment by jguddas** [comment]
I prefer the pointy nose over the round nose version.

**Comment by karsa-mistmere** [comment]
> I prefer the pointy nose over the round nose version.

Good idea, since our regular `plane` also has a somewhat pointy nose. I've updated the icons within the PR accordingly.
